ntpclient/ntpclient.go

```diff
@@ -55,5 +57,6 @@ func NTPClient(pool []string) (time.Time, error) {
 
 		return time.Unix(int64(secs), nanos), nil
 	}
-	return time.Unix(0, 0), errors.New("no valid time servers")
+	log.Warnln(log.Global, "No valid ntp server found returning current system time")
```

Comment:
Please use log.TimeMgr here, and just a basic grammar fix:

`No valid NTP servers found, using current system time`